src/java/org/apache/cassandra/schema/SchemaKeyspace.java

```diff
@@ -67,7 +67,7 @@
  * system_schema.* tables and methods for manipulating them.
  *
  * Please notice this class is _not_ thread safe and all methods which reads or updates the data in schema keyspace
- * should be accessed only from the implementation of {@link SchemaUpdateHandler} in synchronized blocks.
+ * should be accessed only from the implementation of {SchemaUpdateHandler} in synchronized blocks.
```

Comment:
It seems Java doc around Schema needs a re-write at more places. There is no `SchemaUpdateHandler` anymore